vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -106,9 +123,23 @@ class Dropdown(VizroBaseModel):
     _validate_options = model_validator(mode="before")(validate_options_dict)
 
     def __call__(self, options):
+        if self.multi:
+            clientside_callback(
+                ClientsideFunction(namespace="dropdown", function_name="update_dropdown_values"),
+                output=[Output(f"{self.id}", "value"), Output(f"{self.id}_checklist_all", "value")],
+                inputs=[
+                    Input(f"{self.id}_checklist_all", "value"),
+                    Input(f"{self.id}", "value"),
+                    State(f"{self.id}", "options"),
+                ],
+            )
```

Comment:
Let's add a trailing comma at the end of outputs and swap the order of outputs like here (just a heads up to swap the order of outputs in the `update_dropdown_values` function too):
```suggestion
        if self.multi:
            clientside_callback(
                ClientsideFunction(namespace="dropdown", function_name="update_dropdown_values"),
                output=[
                    Output(f"{self.id}_checklist_all", "value"),
                    Output(f"{self.id}", "value"), 
                ],
                inputs=[
                    Input(f"{self.id}_checklist_all", "value"),
                    Input(f"{self.id}", "value"),
                    State(f"{self.id}", "options"),
                ],
            )
```

Reply:
Is there a reason for adding a trailing comma?

Reply:
When you add a trailing comma, lint tools (like `black`) will never "squeeze" the list elements into a single line. The trailing comma is the sign to the `black` to leave every list element in its own line. 

There was the discussion a while ago about that which you can find -> https://github.com/mckinsey/vizro/pull/292.